easybuild/easyconfigs/i/IPython/IPython-3.2.3-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,85 @@
+easyblock = 'Bundle'
+
+name = 'IPython'
+version = '3.2.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://ipython.org/index.html'
+description = """IPython provides a rich architecture for interactive computing with:
+ Powerful interactive shells (terminal and Qt-based).
+ A browser-based notebook with support for code, text, mathematical expressions, inline plots and other rich media.
+ Support for interactive data visualization and use of GUI toolkits.
+ Flexible, embeddable interpreters to load into your own projects.
+ Easy to use, high performance tools for parallel computing."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+dependencies = [
+    ('Python', '2.7.11'),
+    ('PyZMQ', '15.2.0', '%s-zmq4' % versionsuffix),
+]
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+
+exts_list = [
+    ('pysqlite', '2.8.1', {
+        'modulename': 'pysqlite2',
+        'source_urls': ['https://pypi.python.org/packages/source/p/pysqlite/'],
+    }),
+    ('requests', '2.9.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/r/requests/'],
+    }),
+    ('Pygments', '2.0.2', {
+        'source_urls': ['https://pypi.python.org/packages/source/P/Pygments/'],
+    }),
+    ('singledispatch', '3.4.0.3', {
+        'source_urls': ['https://pypi.python.org/packages/source/s/singledispatch/'],
+    }),
+    ('backports.ssl_match_hostname', '3.5.0.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/b/backports.ssl_match_hostname/'],
+    }),
+    ('certifi', '2015.11.20.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/c/certifi/'],
+    }),
+    ('backports_abc', '0.4', {
+        'source_urls': ['https://pypi.python.org/packages/source/b/backports_abc/'],
+    }),
+    ('tornado', '4.3', {
+        'source_urls': ['https://pypi.python.org/packages/source/t/tornado/'],
+    }),
+    ('Jinja2', '2.8', {
```

Comment:
please include `MarkupSafe-0.23` above the Jinja2 entry, apparently it's a (new) dependency for Jinja2

the auto-download/install of MarkupSafe as a dependency for Jinja2 failed in the test reports because the Python install tool has problem with directories that hav `[...]` in the path...